docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
       OPEN_DISCUSSIONS_REDDIT_VALIDATE_SSL: 'False'
       OPEN_DISCUSSIONS_REDDIT_CLIENT_ID: od_client_id
       OPEN_DISCUSSIONS_REDDIT_SECRET: od_client_secret
+      OPEN_DISCUSSIONS_FEATURE_DEFAULT: 'True'
```

Comment:
Missing an `s` in `FEATURE`, should be `OPEN_DISCUSSIONS_FEATURES_DEFAULT`.
Also value should be `${OPEN_DISCUSSIONS_FEATURES_DEFAULT:-False}` instead of `'True'`?

Reply:
@mbertrand we want this to default to `True` for development environments, which are the only ones that use docker-compose.yml. If individual devs was to opt out of that, they can override it in `.env`. I'll fix the name though.